### ISSUE TITLE
Add programming ligatures

### ISSUE THREAD

**Issue by BenvdStouwe** [shannpersand/comic-shanns#2]
Will not use without them. 
See [this page](https://www.hanselman.com/blog/MonospacedProgrammingFontsWithLigatures.aspx).

**Comment by Meligy** [comment]
Please if you ever do that, keep a version available for those of us who may find ligatures confusing and/or anti the clarity that a monospace font gives.

**Comment by 1234567-glitch** [comment]
@Meligy may be able?

**Comment by hiringManager** [comment]
Please add ligatures but make them various unrelated WingDings to preserve clarity.

**Comment by Roshinator** [comment]
The hard part will be getting comic sans style symbols for that, but if someone can make it happen I will switch all my IDEs/text editors to it.

**Comment by kilobyte** [comment]
Please note that, besides some people's dislike on both aesthetic and practical grounds, a lot of display libraries handle such ligatures poorly.  If we're lucky the ligatures are simply ignored, but if the library *tries* to support them but does something wrong, the font becomes unusable.

This said, I'm not going to stop anyone from making a ligated version, just that the plain version would need to be shipped separately.

**Comment by ThomasRettig** [comment]
I'm all for thoughtful ligatures _(eg. those that modify the whitespace between ≥2 characters while not changing the overall glyph width, refer to [JetBrains Mono](https://github.com/JetBrains/JetBrainsMono))_, but, similar to @kilobyte, I don't like fanciful, over-the-top ones, such as Fira Code, whose design concept is well-meant, but unfortunately led astray by gimmicky non-functional ligatures.

Not only do monospace font ligatures adversely impact the readability of code, they are also uninspired, banal, and ultimately defeat the raison d'etre of monospaced fonts: ensuring that every glyph is clear and unambiguous.

As such, I feel that the addition of programming ligatures to this font will be a highly subjective and thorny issue to tackle; for one, I'm sure some would welcome it, but overall, I am not convinced by this idea, and even if ligatures are pursued, a non-ligated version of the font should be made default. Matthew Butterick raises some good points on this matter (read this [insightful article](https://practicaltypography.com/ligatures-in-programming-fonts-hell-no.html)), and I think hardcore monospaced ligature enthusiasts could learn a thing or two from it.

**Comment by Roshinator** [comment]
Of course, I don't know of any ligated font that doesn't have a pure monospaced option. It's 100% necessary for places like terminals where ligatures don't play nice with the terminal program or for people who don't like it.

**Comment by Vishrut19** [comment]
Please font ligatures into this font like Fira Code and Cascadia Code as without font ligatures it is very difficult to use.

**Comment by diantongren** [comment]
> The hard part will be getting comic sans style symbols for that, but if someone can make it happen I will switch all my IDEs/text editors to it.

Maybe you can take a look at this font.
![Image](https://github.com/user-attachments/assets/f3315429-c072-4d4c-a642-cd8940580f63)
- [Toshi Omagari | Comic Code](https://tosche.net/fonts/comic-code)
